src/store/Businesses.js

```diff
@@ -1,29 +1,23 @@
-import { observable, action, decorate } from 'mobx';
+import { observable, action } from 'mobx';
 import api from '../utils/apis';
 
+
 class Businesses {
```

Comment:
I think BusinessStore class name would be better. The store will have all the business logic so you can have an observable in here called businesses that will be an array of business objects